streaming_jupyter_integrations/cast_utils.py

```diff
@@ -0,0 +1,40 @@
+import string
+from random import choice
+
+from pyflink.table import Table, TableEnvironment
+from pyflink.table.types import DataType, LocalZonedTimestampType
+
+
+def cast_timestamp_ltz_to_string(tenv: TableEnvironment, table: Table) -> 'Table':
+    if not __contains_timestamp_ltz_fields(table):
+        return table
+    else:
+        return __cast_timestamp_ltz_to_string(tenv, table)
+
+
+def __contains_timestamp_ltz_fields(table: Table) -> bool:
+    table_schema = table.get_schema()
+    return any(__is_timestamp_ltz(t) for t in table_schema.get_field_data_types())
+
+
+def __is_timestamp_ltz(field_type: DataType) -> bool:
+    return isinstance(field_type, LocalZonedTimestampType)
+
+
+def __cast_timestamp_ltz_to_string(tenv: TableEnvironment, table: Table) -> 'Table':
```

Comment:
Just a proposal :smile_cat: Perhaps `__add_string_cast_for_timestamp_ltz_fields_in_table` would be a more adequate name for the logic of this function?

Reply:
Hmm, IMO the name proposed is a bit too long :P

Reply:
:) Besides that LGTM!